lib/player.rb

```diff
@@ -0,0 +1,19 @@
+class Player
+   #attr_reader allows accees to name and deck
+  attr_reader :name, #attr_reader = methods for class
+              :deck
+
+    def initialize(name, deck) #parameters
+      @name = name   #instance variables
+      @deck = deck
+    end
+
+     def has_lost?
+          if @deck.cards.length > 0
+            false
+          else
+            true
+
+      end
```

Comment:
Keep an eye on your blank spaces and indentation.
```
if @deck.cards.length > 0
  false
else
  true
end
```
Alternatively, you could accomplish this conditional logic on one line and get a boolean return value with something like
```
def has_lost?
  @deck.cards.empty?
end
```